### PR TITLE
refine prompt prefix output

### PR DESCRIPTION
**Pull request by IANTHEREAL**

### CODE REVIEW

File: langchain/experimental/autonomous_agents/autogpt/prompt.py [hunk 1]
```diff
@@ -20,11 +20,9 @@ class AutoGPTPrompt(BaseChatPromptTemplate, BaseModel):
     send_token_limit: int = 4196
 
     def construct_full_prompt(self, goals: List[str]) -> str:
-        prompt_start = """Your decisions must always be made independently 
-            without seeking user assistance. Play to your strengths 
-            as an LLM and pursue simple strategies with no legal complications. 
-            If you have completed all your tasks, 
-            make sure to use the "finish" command."""
+        prompt_start = """Your decisions must always be made independently without seeking user assistance.
+Play to your strengthsas an LLM and pursue simple strategies with no legal complications.
```
Comment (dev2049) [inline langchain/experimental/autonomous_agents/autogpt/prompt.py:24]: strengths as

File: langchain/experimental/autonomous_agents/autogpt/prompt.py [hunk 2]
```diff
@@ -72,4 +70,5 @@ def format_messages(self, **kwargs: Any) -> List[BaseMessage]:
         messages: List[BaseMessage] = [base_prompt, time_prompt, memory_message]
         messages += historical_messages
         messages.append(input_message)
+        print(messages)
```
Comment (dev2049) [inline langchain/experimental/autonomous_agents/autogpt/prompt.py:73]: is this intentional?

Comment (vowelparrot) [reply langchain/experimental/autonomous_agents/autogpt/prompt.py:73]: Ought to be removed

Comment (IANTHEREAL) [reply langchain/experimental/autonomous_agents/autogpt/prompt.py:73]: my carelessness

File: langchain/experimental/autonomous_agents/autogpt/prompt.py (hunk 1 above)
Comment (IANTHEREAL) [reply langchain/experimental/autonomous_agents/autogpt/prompt.py]: addressed

**Pull request merged**